spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -58,7 +58,7 @@ module ::ApplicationHelper; end
         end
 
         it 'includes the application helper' do
-          group = RSpec::Core::Example.describe 'bars/new.html.erb'
+          group = RSpec::Core::ExampleGroup.describe 'bars/new.html.erb'
```

Comment:
This is interesting.

Reply:
I mean... I'm not sure theres a difference :joy: